content/section-2/lesson-9-variables-and-values.md

```diff
@@ -331,4 +331,4 @@ start writing some interesting applications. We can now:
 - Destructure sequences and maps
 
 [^1]: The official [Destructuring in Clojure guide](https://clojure.org/guides/destructuring) is an excellent reference
-[^2]: This is a tongue in cheek reference to Phil Karlton's famous quote, "There are only two hard things in Computer Science: cache invalidation and naming things."
```

Comment:
Sorry, I don't know what happened here: I did not intend to change this last line :-(